go/citogo/s3.go

```diff
@@ -64,33 +64,45 @@ func awsStringSign4(key string, date string, region string, service string, toSi
 
 }
 
-// s3 put without the dependencies, adopted from this shell script:
-//
-//  https://superuser.com/questions/279986/uploading-files-to-s3-account-from-linux-command-line
-//
 func s3put(src io.Reader, bucket string, name string) (string, error) {
 	buf, err := gzipSource(src)
 	if err != nil {
 		return "", err
 	}
-
 	randSuffix, err := randString()
 	if err != nil {
 		return "", err
 	}
 	name += "-" + randSuffix + ".gz"
 
+	_, err = awsCall("s3", bucket+".s3.amazonaws.com", name, "PUT", buf)
+	if err != nil {
+		return "", err
+	}
+	where := fmt.Sprintf("(fetch with: `curl -s -o - https://%s.s3.amazonaws.com/%s | zcat -d`)", bucket, name)
```

Comment:
didn't you change this to make it easier to copy/paste?

Reply:
added back, not sure where it got lost